app/digitizingcontroller.cpp

```diff
@@ -193,3 +193,23 @@ QgsQuickFeatureLayerPair DigitizingController::lineFeature()
 
   return QgsQuickFeatureLayerPair(f, featureLayerPair().layer());
 }
+
+QgsPoint DigitizingController::mapCoords(QgsQuickFeatureLayerPair pair)
+{
+    if ( !pair.layer() )
+      return QgsPoint();
+
+    QgsPointXY res = mMapSettings->mapSettings().layerToMapCoordinates(pair.layer(), QgsPoint(pair.feature().geometry().asPoint()));
+    return QgsPoint(res);
+}
+
+QgsQuickFeatureLayerPair DigitizingController::changePointGeometry(QgsQuickFeatureLayerPair pair, QgsPoint point)
+{
+    QgsPointXY layerPoint = mMapSettings->mapSettings().mapToLayerCoordinates(pair.layer(), QgsPointXY(point.x(), point.y()));
+    QgsPoint* mapPoint = new QgsPoint( layerPoint );
```

Comment:
It is confusing to call it mapPoint when in fact it is using layer coordinate system.... maybe use layerPointXY and layerPoint variables?